[2022-07-22 13:00:20,942] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-25T00:00:00+00:00 [queued]>
[2022-07-22 13:00:21,003] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-25T00:00:00+00:00 [queued]>
[2022-07-22 13:00:21,006] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:21,012] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 13:00:21,014] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:21,086] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-25 00:00:00+00:00
[2022-07-22 13:00:21,099] {standard_task_runner.py:52} INFO - Started process 813 to run task
[2022-07-22 13:00:21,117] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-25T00:00:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpqjj8cbj7', '--error-file', '/tmp/tmp_u4itulz']
[2022-07-22 13:00:21,123] {standard_task_runner.py:80} INFO - Job 54: Subtask t2
[2022-07-22 13:00:21,200] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 13:00:21,383] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-25T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 13:00:21,526] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-25T00:00:00+00:00
[2022-07-22 13:00:21,530] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-22 13:00:21,599] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220625T000000, start_date=20220722T130020, end_date=20220722T130021
[2022-07-22 13:00:21,701] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 13:00:21,808] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:18:05,161] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-25T00:00:00+00:00 [queued]>
[2022-07-25 09:18:05,181] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-25T00:00:00+00:00 [queued]>
[2022-07-25 09:18:05,183] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:05,184] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:18:05,186] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:05,231] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-25 00:00:00+00:00
[2022-07-25 09:18:05,238] {standard_task_runner.py:52} INFO - Started process 777 to run task
[2022-07-25 09:18:05,248] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-25T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpb2uu559d', '--error-file', '/tmp/tmpvgew63tl']
[2022-07-25 09:18:05,282] {standard_task_runner.py:80} INFO - Job 55: Subtask t2
[2022-07-25 09:18:05,367] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:18:05,572] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-25T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:18:05,834] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-25T00:00:00+00:00
[2022-07-25 09:18:05,836] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 09:18:05,900] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220625T000000, start_date=20220725T091805, end_date=20220725T091805
[2022-07-25 09:18:05,960] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:18:06,029] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 12:26:56,732] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-25T00:00:00+00:00 [queued]>
[2022-07-25 12:26:56,812] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-25T00:00:00+00:00 [queued]>
[2022-07-25 12:26:56,819] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 12:26:56,820] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 12:26:56,822] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 12:26:56,940] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-25 00:00:00+00:00
[2022-07-25 12:26:56,969] {standard_task_runner.py:52} INFO - Started process 848 to run task
[2022-07-25 12:26:56,998] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-25T00:00:00+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpjnvegio3', '--error-file', '/tmp/tmpfz30wrhb']
[2022-07-25 12:26:57,020] {standard_task_runner.py:80} INFO - Job 125: Subtask t2
[2022-07-25 12:26:57,125] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 12:26:57,510] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-25T00:00:00+00:00 [running]> on host 929e6a3728d4
[2022-07-25 12:26:58,114] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-25T00:00:00+00:00
[2022-07-25 12:26:58,116] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 12:26:58,327] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220625T000000, start_date=20220725T122656, end_date=20220725T122658
[2022-07-25 12:26:58,464] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 12:26:58,638] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
